### PR TITLE
Formatting fixes on Observability use case guide

### PR DESCRIPTION
**Pull request by Blargian**
## Summary
Some small formatting fixes to Observability use case guide pages. Mainly for the query result tables looking odd:

image

The spacing changes look strange in Github code view in some places but when I build docs locally they look correct (see below), so we should check on Vercel deployment.

image

## Checklist
- [X] Delete items not relevant to your PR
- [X] URL changes should add a redirect to the old URL via https://github.com/ClickHouse/clickhouse-docs/blob/main/docusaurus.config.js
- [X] If adding a new integration page, also add an entry to the integrations list here: https://github.com/ClickHouse/clickhouse-docs/blob/main/docs/en/integrations/index.mdx

### CODE REVIEW

File: docs/en/use-cases/observability/integrating-opentelemetry.md [hunk 1]
```diff
@@ -600,9 +600,9 @@ By default, each insert sent to ClickHouse causes ClickHouse to immediately crea
 
 By default, inserts into ClickHouse are synchronous and idempotent if identical. For tables of the merge tree engine family, ClickHouse will, by default, automatically [deduplicate inserts](https://clickhouse.com/blog/common-getting-started-issues-with-clickhouse#5-deduplication-at-insert-time). This means inserts are tolerant in cases like the following:
 
-1. If the node receiving the data has issues, the insert query will time out (or get a more specific error) and not receive an acknowledgment.
-1. If the data got written by the node, but the acknowledgement can't be returned to the sender of the query because of network interruptions, the sender will either get a time-out or a network error.
-
+- (1) If the node receiving the data has issues, the insert query will time out (or get a more specific error) and not receive an acknowledgment.
+- (2) If the data got written by the node, but the acknowledgement can't be returned to the sender of the query because of network interruptions, the sender will either get a time-out or a network error.
+
```
Comment (Blargian) [inline docs/en/use-cases/observability/integrating-opentelemetry.md:605]: This change is due to some weird rendering issue which causes the paragraph after the numbered list to bunch up close to the list: 
Screenshot from 2024-11-09 12-07-53

For some reason I cannot get that paragraph to render nicely with a space after the list so I eventually resorted to using `- (1)`, `-(2)`. Looks better although would be nicer without the bullets:

Screenshot from 2024-11-09 12-07-25

File: docs/en/use-cases/observability/schema-design.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ We recommend users always create their own schema for logs and traces for the fo
...
+**Important:** While users are encouraged to extend and modify their schema to achieve optimal compression and query performance, they should adhere to the OTel schema naming for core columns where possible. The ClickHouse Grafana plugin assumes the existence of some basic OTel columns to assist with query building e.g. Timestamp and SeverityText. The required columns for logs and traces are documented here [[1]](https://grafana.com/developers/plugin-tools/tutorials/build-a-logs-data-source-plugin#logs-data-frame-format)[[2]](https://grafana.com/docs/grafana/latest/explore/logs-integration/) and [here](https://grafana.com/docs/grafana/latest/explore/trace-integration/#data-frame-structure), respectively. You can choose to change these column names, overriding the defaults in the plugin configuration.
```
Comment (Blargian) [inline docs/en/use-cases/observability/schema-design.md:21]: added space after and and the link for `"here"`

File: docs/en/use-cases/observability/schema-design.md [hunk 3]
```diff
@@ -1112,8 +1112,10 @@ We've effectively reduced the number of rows here from 10m (in `otel_logs`) to 1
 
 Since the merging of rows is asynchronous, there may be more than one row per hour when a user queries. To ensure any outstanding rows are merged at query time, we have two options:
 
-Use the [`FINAL` modifier](/en/sql-reference/statements/select/from#final-modifier) on the table name. We did this for the count query above.
-Aggregate by the ordering key used in our final table i.e. Timestamp and sum the metrics. Typically, this is more efficient and flexible (the table can be used for other things), but the former can be simpler for some queries. We show both below:
+- Use the [`FINAL` modifier](/en/sql-reference/statements/select/from#final-modifier) on the table name (which we did for the count query above).
+- Aggregate by the ordering key used in our final table i.e. Timestamp and sum the metrics. 
+
```
Comment (Blargian) [inline docs/en/use-cases/observability/schema-design.md:1117]: Probably a nit, just seemed a little easier to read if we split the second point.

File: docs/en/use-cases/observability/schema-design.md [hunk 4]
```diff
@@ -1311,7 +1313,7 @@ WHERE TraceId != ''
 GROUP BY TraceId
 ```
 
-The view effectively ensures the table `otel_traces_trace_id_ts` has the minimum and maximum Timestamp for the trace. This table, ordered by `TraceId`, allows these timestamps to be retrieved efficiently. These timestamp ranges can, in turn, be used when querying the main `otel_traces` table. More specifically, when retrieving a trace by its id, Grafana uses the following query:
+The view effectively ensures the table `otel_traces_trace_id_ts` has the minimum and maximum timestamp for the trace. This table, ordered by `TraceId`, allows these timestamps to be retrieved efficiently. These timestamp ranges can, in turn, be used when querying the main `otel_traces` table. More specifically, when retrieving a trace by its id, Grafana uses the following query:
```
Comment (Blargian) [inline docs/en/use-cases/observability/schema-design.md:1316]: This is referring to max and min timestamps in general I think, not some specific column name called `Timestamp`?

**Comment by gingerwizard** [comment]
Rendering LGTM. Merging.

**Pull request merged**